src/tutorials/volume-snapshots-how-to-create-and-use-data-clones-in-okteto.mdx

```diff
@@ -0,0 +1,311 @@
+---
+title: Volume Snapshots: How to Create a Development Environment with Realistic Data in Okteto
+description: Learn how to create a data clone of an application’s database, and how to use it when creating a new Development Environment.
+---
+
+import Image from '@theme/Image';
+
+Data cloning allows you to create an independent, identical copy of an application’s database, making it easy to test with realistic data.
+Using Okteto Volume Snapshots, you can quickly create development environments that include production or staging data—without manually copying databases or waiting for CI/CD pipelines.
+
+:::tip
+Volume Snapshots are also useful for loading test data when running tests in Okteto Test. If your tests require a database with preloaded data, you can configure your test environment to restore from a snapshot instead of initializing an empty database. This ensures consistency across test runs and reduces setup time.
+:::
+
+In this tutorial, you'll:
+✅ deploy an application to Okteto
+✅ learn how to create a data clone of the application's database
+✅ and then learn how to use it when creating a new Development Environment
+
+## Prerequisites
+
+In order to follow this tutorial, you should have:
+
+-   A **Kubernetes cluster with Okteto** installed
+-   Familiarity with **Kubernetes manifests**
+-   [kubectl](https://kubernetes.io/docs/tasks/tools/) installed on your machine
```

Comment:
you also need to create the volumesnapshotclass that we use (https://kubernetes.io/docs/concepts/storage/volume-snapshot-classes/). I believe we have docs for this, I don't think we create one by default when you install Okteto (which maybe we should?)